src/components/shared/paginator/Paginator.tsx

```diff
@@ -0,0 +1,119 @@
+import { range } from 'lib/helpers'
+import React from 'react'
+import styled, { css } from 'react-emotion'
+import { Button, Table } from 'semantic-ui-react'
```

Comment:
May be a lot to ask for, but maybe we can try to use hedvig-ui buttons
```suggestion
import { Button, ButtonGroup } from 'shared/hedvig-ui'
```

Reply:
Good point!